stats/src/main/java/com/facebook/airlift/stats/cardinality/DenseHll.java

```diff
@@ -22,21 +22,21 @@
 import io.airlift.slice.Slice;
 import org.openjdk.jol.info.ClassLayout;
 
-import javax.annotation.concurrent.NotThreadSafe;
-
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
 
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkState;
 import static com.facebook.airlift.stats.cardinality.Utils.alpha;
 import static com.facebook.airlift.stats.cardinality.Utils.computeIndex;
 import static com.facebook.airlift.stats.cardinality.Utils.computeValue;
 import static com.facebook.airlift.stats.cardinality.Utils.linearCounting;
 import static com.facebook.airlift.stats.cardinality.Utils.numberOfBuckets;
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkState;
 
-@NotThreadSafe
+/**
```

Comment:
IDK why they did this. I'll let someone more familiar with Java decide what's better

Reply:
This was changed as part of https://github.com/airlift/airlift/commit/1eb30dc9a59d3e184340855a9177c949ec01ac4f. 

It looks like the background is that javax.annotations was moved and renamed to jakarta.annotations (rename was due to some licensing disagreement between oracle and eclipse see https://github.com/jakartaee/rest/issues/760).  But Jakarta.annotations doesn't have the NotThreadSafe annotation (see discussion here: https://github.com/jakartaee/common-annotations-api/pull/91).  So seems like this change was basically part of upgrading that library.